pipeline.yaml

```diff
@@ -95,6 +121,17 @@ jobs:
       inlineScript: |
         az aks get-credentials --resource-group $(aksResourceGroup) --name $(aksClusterName) --admin
 
+  - task: AzureCLI@1
```

Comment:
use `Bash@3` task here if you don't need and `az` commands

Reply:
done for all